app/src/main/java/com/fstyle/androidtrainning/screen/main/MainContract.java

```diff
@@ -1,5 +1,9 @@
 package com.fstyle.androidtrainning.screen.main;
 
+import com.fstyle.androidtrainning.data.remote.service.config.LastFmApi;
+import com.fstyle.androidtrainning.model.Track;
+import java.util.List;
```

Comment:
ERROR: Unused import: java.util.List. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports